internal/utils/main.go

```diff
@@ -231,6 +231,7 @@ func AddLPOperations(txMeta []xdr.OperationMeta, AssetA, AssetB xdr.Asset) []xdr
 // AddCommonFlags adds the flags common to all commands: start-ledger, end-ledger, stdout, and strict-export
 func AddCommonFlags(flags *pflag.FlagSet) {
 	flags.Uint32P("end-ledger", "e", 0, "The ledger sequence number for the end of the export range")
+	flags.String("end-timestamp", "", "The ledger sequence number for the end of the export range")
```

Comment:
Curious why start-timestamp is a core and archive flag vs end-timestamp is a common flag?